install.md

```diff
@@ -149,6 +149,12 @@ sudo apt-get install \
   uidmap
 ```
 
+On openSUSE Leap:
```

Comment:
Is this bound to any version, like Leap > 15? Do we want to add a note about Tumbleweed as well? :)

Reply:
Done. :)